editor/i18n/zh/localization.js

```diff
@@ -904,9 +908,13 @@ module.exports = {
             label: "自定义渲染管线（实验）",
             description: "启用自定义渲染管线。",
         },
+        websocket: {
+            label: "WebSocket",
+            description: "对原生启用 WebSocket。其中iOS/macOS使用SocketRocket实现, Android使用OkHttp实现, Windows使用libwebsockets实现。",
+        },
         websocket_server: {
             label: "WebSocket Server",
-            description: "对原生启用 WebSocket Server。",
+            description: "对原生启用 WebSocket Server。注意: 必须同时启用 WebSocket。",
```

Comment:
Can automatically enable WebSocket if WebSocket Server is enabled?

Reply:
At present, there is no such mechanism on the editor. This dependency will be provided by cc.config.json. @yanOO1497 said that this part needs engine determined the fields and editor could implement this mechanism.

Reply:
track in issue: https://github.com/cocos/3d-tasks/issues/14052